lib/fb/post.rb

```diff
@@ -2,25 +2,43 @@
 # @see http://www.rubydoc.info/gems/Fb/
 module Fb
   # Fb::Post reprensents a Facebook post. Post provides getters for:
-  #   :id, :title, :url, :created_time, and :type.
+  #   :id, :url, :created_at, :type, :message, and :length.
   class Post
+
     # @option [String] the post’s unique ID.
     attr_reader :id
 
-    # @option [String] the post’s type.
-    attr_reader :type
+    # @option [String] the post’s permanent url.
+    attr_reader :url
 
     # @option [Time] the post’s creation time.
     attr_reader :created_at
 
+    # @option [String] the post’s type.
+    attr_reader :type
+
+    # @option [String] the post’s message.
+    attr_reader :message
+
+    # @option [String] the attached video's length if type video.
+    attr_reader :length
+
     # @param [Hash] options the options to initialize an instance of Fb::Post.
-    # @option [String] :id the post id.
-    # @option [String] :type the post’s type.
-    # @option [String] :created_time the post’s creation time in iso8601 format.
+    # @option [String] :id The post id.
+    # @option [String] :message The status message in the post or post story.
+    # @option [String] :url URL to the permalink page of the post.
+    # @option [String] :created_time The time the post was initially published.
+    # @option [String] :type A string indicating the object type of this post.
+    # @option [String] :length The length of the attached video.
```

Comment:
@S1v4 this is not correct, you are getting properties, not length.

Also it would be helpful to add an `@example` to show the format of all these options